.github/workflows/build-for-request.yml

```diff
@@ -0,0 +1,44 @@
+name: build for request
+
+on:
+  push:
+  pull_request:
+    branches:
+      - master
```

Comment:
We updated the *default* branch from `master` to `main`. Could you update it please?

Reply:
Done.